roofit/roofitcore/src/RooProdPdf.cxx

```diff
@@ -439,7 +431,6 @@ void RooProdPdf::initializeFromCmdArgList(const RooArgSet& fullPdfSet, const Roo
 
 RooProdPdf::~RooProdPdf()
 {
-  _pdfNSetList.Delete() ;
```

Comment:
This is the indication that the list on disc doesn't own the objects ...

Reply:
Right, that's a good hint!